.github/workflows/validate_package_noop.yml

```diff
@@ -0,0 +1,14 @@
+name: Validate Frictionless Data Package Schema for Spec
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '!spec/tides.spec.json$'
+
+jobs:
+  validate_data_package:
+    runs-on: ubuntu-latest
+    steps:
+      - run: 'echo "no validation required"'
```

Comment:
add comment on why, i.e.

`no changes to spec/tides.spec.json: no validation required`